core/scheduler/service/replay_service.go

```diff
@@ -39,12 +38,17 @@ type ReplayValidator interface {
 	Validate(ctx context.Context, replayRequest *scheduler.Replay, jobCron *cron.ScheduleSpec) error
 }
 
+type ReplayExecutor interface {
+	Execute(replayID uuid.UUID, jobTenant tenant.Tenant, jobName scheduler.JobName)
+}
+
 type ReplayService struct {
 	replayRepo ReplayRepository
 	jobRepo    JobRepository
 	runGetter  SchedulerRunGetter
 
 	validator ReplayValidator
+	worker    ReplayExecutor
```

Comment:
should we name it to `executor`? because the type is `ReplayExecutor` not `ReplayWorker`

Reply:
updated to `executor`